design/cdn-edge.md

```diff
@@ -6,62 +6,62 @@ On this diagram the arrow points in the direction of who instantiated the reques
 
-In terms of network hops.
+In terms of network Hops.
 - Layer 1 (cache miss)
 - Layer 2 (cache miss)
-- Layer 2 (cache miss & no direct connecton)
+- Layer 2 (cache miss & no direct connection)
 - Origin (source of truth)
 
-If layer 2 in hop 2 has a direct connection to the origin we will route directly to hop 4 (skipping over hop 3).
+If Layer 2 in Hop 2 directly connects to the origin, we will route directly to Hop 4 (skipping over Hop 3).
 
 ## Control
 
-The control servers will act as the state of the system. They will contain all the information about what routes exist in the network. They will also act as a authentication and state management server for the layer 2 servers which will in turn act as a manager for layer 1 servers.
+The control servers will act as the state of the system. They will contain all the information about what routes exist in the network. They will also serve as an authentication and state management server for the Layer 2 servers, which will act as a manager for Layer 1 servers.
 
-Control servers are completely isolated from the request lifecycle and therefore do not a large bandwidth pipe.
+Control servers are entirely isolated from the request lifecycle and therefore do not have a large bandwidth pipe.
 
 ## Origin 
 
-Origin servers will be the source of truth for all the data. If all layers above cache miss the origin will provide a fresh copy of the data for the layers above to use.
+Origin servers will be the source of truth for all the data. If all Layers above cache miss, the origin will provide a fresh copy of the data for the Layers above to use.
 
 ## FAQ
 
 ### __What is this?__
 
-_You may have realized this is a very simple CDN setup similar to that of Cloudflare or Fastly or any other large CDN as a service provider._
+_You may have realized this is a straightforward CDN setup similar to that of Cloudflare or Fastly, or any other large CDN as a service provider._
 
-### __Why build your own if there are very cheap and cost effective solutions that exist?__
+### __Why build your own if very cheap and cost-effective solutions exist?__
 
-_Well, they are not very cheap when you start looking at them at scale. They become extremely expensive and add up very fast._
+_Well, they are not very cheap when looking at them at scale. They become costly and add up very fast_
 
 ### __What makes this solution special?__
 
-_This edge cache will be designed to taylor to specific needs of a video edge cache, and will have unique features which improve performance that other providers do not have._
+_This edge cache is tailored to the specific requirements of a video edge cache and has unique features that improve performance that other vendors do not offer._
```

Comment:
Not sure "vendors" is the right word, maybe "platforms"?

Reply:
vendors is correct since this is a CDN.